src/modules/client/mixins/planningActionMixin.js

```diff
@@ -37,7 +37,10 @@ export const planningActionMixin = {
       newEvent: {
         type: { required },
         dates: {
-          startDate: { required },
+          startDate: {
+            required,
+            maxDate: this.getStoppingDate(this.newEvent) ? maxDate(this.getStoppingDate(this.newEvent)) : '',
```

Comment:
est-ce que `maxDate(this.getStoppingDate(this.newEvent) || '')` fonctionne ?

Reply:
malheureusement non maxDate('') n'est pas égal à ''